.release-please-manifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  ".": "1.21.1",
-  "packages/api-explorer": "0.9.26",
+  ".": "2.0.0",
```

Comment:
ok, I finally figured out why the weird `1.22.0` versions were showing up:

release-please as a BUNCH of methods for overriding what the version for the next release of a package should be and using the string `Release-As: x.y.z` in a commit message is one of them. @drstrangelooker when you squash merged https://github.com/looker-open-source/sdk-codegen/pull/959 there was a commit in that list that had the title `Release-As: 1.22.0` and it made it into the finally commit message of the [squashed merge](https://github.com/looker-open-source/sdk-codegen/commit/de651b373978fd3689429c0cbdb364a2820fe211) and any package with a file in that commit now gets that set.

For the version of release-please we're on, this "commit release-as" method of setting the version overrides everything else. I don't think there's an "out-of-the-box" way to fix this so I've run release-please locally on my mac and commented out the "commit release-as" block. As a result I'm seeing much better version numbers now.

Once https://github.com/looker-open-source/sdk-codegen/pull/985 is merged in I'll re-do it again and then we'll merge it in